### ISSUE TITLE
Using fine-tuned models on HMDB but got unexpected classes output

### ISSUE THREAD

**Issue by hiankun** [kenshohara/video-classification-3d-cnn-pytorch#35]
Are models such as `resnext-101-kinetics-hmdb51_split1.pth`, `resnext-101-64f-kinetics-hmdb51_split1.pth`, and `resnet-101-kinetics-hmdb51_split1.pth` fine-tuned on HMDB data sets? If it's the case, should they output action classes within the 51 classes of HMDB?

I downloaded the three pre-triained models mentioned above and ran commands as the follows (using the first model as the example):
```
python main.py --input input --video_root ../test_videos/ --output output.json --model_name resnext --model ../pretrained_models/resnext-101-kinetics-hmdb51_split1.pth --model_depth 101 --resnet_shortcut B --batch_size 8
```
but the output (generated video) always show actions named `belly dancing`, `bending metal`, `braiding hair`,... and so on. All the actions are not from [the 51 classes of HMDB](http://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/#overview).

Did I do something wrong when I was using the pre-trained models? Or my understanding on the fine-tuned models is not correct?

**Comment by hiankun** [comment]
It turns out that I misused the class label file (`class_names_list`)...
